### PR TITLE
Con 2668

### PR DESCRIPTION
**Pull request by hkf-tech**

### CODE REVIEW

File: src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipCompletedFunction.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        public ApprenticeshipCompletedFunction(IForecastingDbContext forecastingDbContext)
+        {
+            _forecastingDbContext = forecastingDbContext;
+        }
+
+        [FunctionName("ApprenticeshipCompleted")]
+        public async Task Run(
+            [NServiceBusTrigger(Endpoint = "SFA.DAS.Fcast.ApprenticeshipCompletedEvent")] ApprenticeshipCompletedEvent message)
+        {
+            var selectedApprenticeship = _forecastingDbContext.Commitment.Where(x => x.ApprenticeshipId == message.ApprenticeshipId).First();
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipCompletedFunction.cs:22]: How do we know first is the only/correct apprenticeship? If we know there is only 1 then we should use Single.

Also what do we do if the apprenticeship does not exist for this event?

File: src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipStoppedFunction.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        public ApprenticeshipStoppedFunction(IForecastingDbContext forecastingDbContext)
+        {
+            _forecastingDbContext = forecastingDbContext;
+        }
+
+        [FunctionName("ApprenticeshipStopped")]
+        public async Task Run(
+            [NServiceBusTrigger(Endpoint = "SFA.DAS.Fcast.ApprenticeshipStopped")] ApprenticeshipStoppedEvent message)
+        {
+            var selectedApprenticeship = _forecastingDbContext.Commitment.Where(x => x.ApprenticeshipId == message.ApprenticeshipId).First();
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipStoppedFunction.cs:25]: Same as above question

File: src/SFA.DAS.Forecasting.Commitments.Functions/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+[assembly: FunctionsStartup(typeof(SFA.DAS.Forecasting.Commitments.Functions.Startup))]
+namespace SFA.DAS.Forecasting.Commitments.Functions
+{
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.Services.AddLogging(logBuilder =>
+            {
+                logBuilder.AddFilter(typeof(Startup).Namespace, LogLevel.Information); // this is because all logging is filtered out by defualt
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Commitments.Functions/Startup.cs:22]: typo in word "default" in comments

File: src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipCompletedFunction.cs (hunk 1 above)
Comment (hkf-tech) [reply src/SFA.DAS.Forecasting.Commitments.Functions/NServicebusTriggerFunctions/ApprenticeshipCompletedFunction.cs]: Made changes,  CON-2700 contains further changes for no apprenticeship found.

**Review by cofaulco — APPROVED** [review]

**Pull request merged**